juju/juju
# OAuth 2.8 credentials support in Juju 2.9

OAuth 2.8 credentials support in 2.9.

This change updates the Juju facades to return Kubernetes credential that a pre 2.9 releases can understand. This has been done through a facade bump for all users of the CloudSpec facade.


## Checklist


## QA steps

1. Deploy a Juju 2.8 controller with a new model and deploy a workload.
2. Upgrade the controller to this PR
3. Check that the workload agent and support components have no errors and are communicating with the Kubernetes api.
4. Check controller logs for errors

## Bug reference

https://bugs.launchpad.net/juju/+bug/1929910
